### DISCUSSION TITLE
请问 Xray(M+H+K+A)+Nginx 配置能否配成偷别人的域名？

### DISCUSSION THREAD

**Discussion by GilHogan** [lxhao61/integrated-examples#229] [Q&A] [answered]
想配一个xray前置，reality偷别人的域名，然后自己的域名还能支持cdn。请问大佬能否实现

**Comment by lxhao61** [comment, marked as answer]
参考 Xray(M+H+K+A)+Nginx 配置来修改不能，参考 Xray(M+F+H+K+A)+Nginx 配置来修改可以、但不推荐。因为 IP 对应两个完全不同网站(域名)，特别 REALITY 所偷域名为知名网站时。
修改方法参考https://github.com/lxhao61/integrated-examples/discussions/222 。

**Reply by GilHogan** [reply]
> 不能。参考 Xray(M+F+H+K+A)+Nginx 修改可以，但不推荐。因为 IP 对应两个完全不同网站(证书)，特别 REALITY 为所偷大网站。 修改方法参考#222 。

ip对应两个完全不同网站(证书)有什么潜在问题吗

**Reply by lxhao61** [reply]
你想呀，一个不知名 IP 对应两个差异很大网站(域名)，不经核查呀。
另外你有域名，真的有必要偷别人网站(域名)？
